# predicted x and u for NMPC
Header header
std_msgs/Float64MultiArray[] x
std_msgs/Float64MultiArray[] u
